deploy/bz1997062-crio-pid-leak/00-osd-cleanup-pids.DaemonSet.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: execpid-cleaner
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      name: execpid-cleaner
+  template:
+    metadata:
+      labels:
+        name: execpid-cleaner
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/infra
```

Comment:
```suggestion
```

Reply:
I think we want to schedule this on any node, not just infra

Reply:
i thought tolerations simply allowed a pod to be scheduled on a node with that taint (as opposed to restricting it to nodes with that taint)

am i misunderstanding?

Reply:
You're not, but this will prevent it from tolerating the master node taint.